geopandas/base.py

```diff
@@ -5003,6 +5003,66 @@ def sample_points(self, size, method="uniform", seed=None, rng=None, **kwargs):
 
         return GeoSeries(result, name="sampled_points", crs=self.crs, index=self.index)
 
+    def build_area(self, node=True):
+        """Creates an areal geometry formed by the constituent linework.
+
+        Builds areas from the GeoSeries that contain linework which represents the edges
```

Comment:
Maybe we should be more explicit here that this happens per row? Because as it reads now, I might also expect to build areas for all line elements of the GeoSeries together (which is not what happens AFAIK. The postgis example in https://postgis.net/docs/ST_BuildArea.html has an `ST_Collect` step in there)

Reply:
Ah, no, actually looking further in the PR at the implementation and the docstring example, my assumption was wrong. This _does_ combine all data. I was just confused about it returning a GeoSeries (naively thinking it would then be element-wise).

So we should maybe clarify the return value, that it are new polgyons formed from the linework of the input and explicitly mention that the values/rows in this returned GeoSeries don't map to certain values in the input?

Reply:
Added a sentence to the end of this paragraph.